packages/storybook/stories/OMBInfo.stories.jsx

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import OMBInfo from '../../react-components/src/components/OMBInfo/OMBInfo';
+import { StoryDocs } from './wc-helpers';
```

Comment:
If we're using this for React components now we may want to have it in a different file for clarity. Btw I think it's pretty cool that we both saw an opportunity to make changes to `StoryDocs` for reusability (#441) - let's try to coordinate those changes so that one of us doesn't have to deal with a ton of merge conflicts 😅

Reply:
Just leaving a comment here after we talked about coordinating changes. I think we can wait to move things into files that aren't named for web components.